TourMate/TourMate/Frontend/Authentication/Services/AuthenticationService.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 protocol AuthenticationService: ObservableObject {
     associatedtype ConcreteAuthenticationService: AuthenticationService
-    static var singleton: ConcreteAuthenticationService { get }
```

Comment:
Rename singleton to shared because singleton is taboo

Reply:
LOL